static/dream_web/index.html

```diff
@@ -8,58 +8,73 @@
-            <option value="192">192</option> <option value="256">256</option>
-            <option value="320">320</option> <option value="384">384</option>
-            <option value="448">448</option> <option value="512">512</option>
-            <option value="576">576</option> <option value="640">640</option>
-            <option value="704">704</option> <option value="768">768</option>
-            <option value="832">832</option> <option value="896">896</option>
-            <option value="960">960</option> <option value="1024">1024</option>
-          </select>
-          <br>
-          <label title="Upload an image to use img2img" for="initimg">Img2Img Init:</label>
-          <input type="file" id="initimg" name="initimg" accept=".jpg, .jpeg, .png">
-          <label title="Set to -1 for random seed" for="seed">Seed:</label>
-          <input value="-1" type="number" id="seed" name="seed">
-          <button type="button" id="reset">&olarr;</button>
-        </fieldset>
-      </form>
-      <div id="about">For news and support for this web service, visit our <a href="http://github.com/lstein/stable-diffusion">GitHub site</a></div>
-    </div>
-    <hr>
-    <div id="results">
-      <div id="no-results-message">
-        <i><p>No results...</p></i>
+    <div id="content-wrapper">
+      <div id="search">
+        <h2 id="header">Stable Diffusion <span>Dream Server</span></h2>
+        <form id="generate-form" method="post" action="#">
+          <fieldset id="fieldset-search">
+            <textarea id="prompt" name="prompt" rows="6" placeholder="Prompt..."></textarea>
+          </fieldset>
+          <fieldset id="fieldset-config">
+            <div id="configs-container">
+                <div class="config-container">
+                  <label for="iterations">N&#176; images:</label>
+                  <input value="1" type="number" id="iterations" name="iterations">
+                </div>
+                <div class="config-container">
+                  <label for="steps">Steps:</label>
+                  <input value="50" type="number" id="steps" name="steps">
+                </div>
+                <div class="config-container">
+                  <label for="cfgscale">Cfg Scale:</label>
+                  <input value="7.5" type="number" id="cfgscale" name="cfgscale" step="any">
+                </div>
+                <div class="config-container">
+                  <label title="Set to multiple of 64" for="width">Width:</label>
+                    <select id="width" name="width" value="512">
+                      <option value="64">64</option> <option value="128">128</option>
+                      <option value="192">192</option> <option value="256">256</option>
+                      <option value="320">320</option> <option value="384">384</option>
+                      <option value="448">448</option> <option value="512" selected="selected">512</option>

```

Comment:
Since the default value is provided by the `value` attribute of the select element, there is no need to add a `selected` attribute here.

Reply:
I think the 512 size is more likely to be used, so I added the `selected` attribute to be the default value.